arviz/tests/test_stats.py

```diff
@@ -117,3 +117,8 @@ def test_waic():
 
     assert_almost_equal(calculated_waic.WAIC, actual_waic, decimal=2)
```

Comment:
This should be lowercase (`calculated_waic.waic`, `calculated_waic.waic_se`). Sorry, this is my fault for copy&pasting from PyMC3.

Reply:
LOL, thanks! (I really should have run the tests locally first).